src/deploy.sh

```diff
@@ -104,7 +104,9 @@ dagster-cloud workspace add-location \
     --location-name "${INPUT_LOCATION_NAME}" \
     --image "${INPUT_REGISTRY}:${INPUT_IMAGE_TAG}" \
     --location-load-timeout 600 \
-    --agent-heartbeat-timeout $AGENT_HEARTBEAT_TIMEOUT
+    --agent-heartbeat-timeout $AGENT_HEARTBEAT_TIMEOUT \
+    --git-url $BRANCH_URL \
```

Comment:
this should be set regardless of whether or not this is branch deployment, yes?

Reply:
yup

Reply:
nope

Reply:
fixed